modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -8,35 +8,114 @@
+
+You can create multiple `KubeletConfig` CRs, as needed, with a limit of ten per cluster. For the first `KubeletConfig` CR, the MCO creates a machine config appended with `kubelet`. With each subsequent CR, the controller creates a new `kubelet` machine config with a numeric suffix. For example, if you have a `kubelet` machine config with a `-2` suffix, the next `kubelet` machine config is appended with `-3`.
+
+If you want to delete the machine configs, you should delete them in reverse order to avoid exceeding the limit. For example, you should delete the `kubelet-3` machine config before deleting the `kubelet-2` machine config.
+
+[NOTE]
+====
+If you have a machine config with a `kubelet-9` suffix, and you create another `KubeletConfig` CR, a new machine config is not created, even if there are fewer than 10 `kubelet` machine configs.
+====
+
+.Example showing a `KubeletConfig` CR
+[source,terminal]
+----
+$ oc get kubeletconfig
+----
+
+[source, terminal]
+----
+NAME                AGE
+set-max-pods        15m
+----
+
+.Example showing a `KubeletConfig` machine config
+[source,terminal]
+----
+$ oc get mc | grep kubelet
+----
+
+[source, terminal]
+----
+...
+99-worker-generated-kubelet-1                  b5c5119de007945b6fe6fb215db3b8e2ceb12511   3.2.0             26m
+...
+----
+
+.Prerequisites
+
+. Obtain the label associated with the static `MachineConfigPool` CR for the type of node you want to configure.
+Perform one of the following steps:
+
+.. View the machine config pool:
++
+[source,terminal]
+----
+$ oc describe machineconfigpool <name>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc describe machineconfigpool worker
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  creationTimestamp: 2019-02-08T14:52:39Z
+  generation: 1
+  labels:
+    custom-kubelet: set-max-pods <1>
+----
+<1> If a label has been added it appears under `labels`.
+
+.. If the label is not present, add a key/value pair:
++
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=set-max-pods
+----
 
 .Procedure
 
-. Run:
+The following procedure is an example to show how to configure the maximum number of pods per node on the worker nodes.
+
+. View the available machine configuration objects that you can select:
 +
 [source,terminal]
 ----
 $ oc get machineconfig
 ----
 +
-This provides a list of the available machine configuration objects you can select. By default, the two kubelet-related configs are `01-master-kubelet` and `01-worker-kubelet`.
+By default, the two kubelet-related configs are `01-master-kubelet` and `01-worker-kubelet`.
 
-. To check the current value of max pods per node, run:
+. Check the current value for the maximum pods per node:
 +
 [source,terminal]
 ----
-# oc describe node <node-ip> | grep Allocatable -A6
+# oc describe node <node_name>
```

Comment:
Was the switch from the earlier `$` prompt to `#` here (and the next instance) intentional?